plugins/module_utils/aci.py

```diff
@@ -320,6 +320,27 @@ def integrate_url(httpapi_url, local_path):
     return {"protocol": parse_url.scheme, "host": parse_url.netloc, "path": local_path}
 
 
+def action_rule_set_comm_spec():
+    return dict(
+        community=dict(type="str"),
+        criteria=dict(type="str", choices=["append", "none", "replace"]),
+    )
+
+
+def action_rule_set_dampening_spec():
+    return dict(
+        half_life=dict(type="int"),
+        max_suppress_time=dict(type="int"),
+        reuse=dict(type="int"),
+        suppress=dict(type="int"),
+    )
+
+def check_all_none_values_dict(x):
```

Comment:
Why do we need this function? can't you just call the `any(x.values())` in the conditional directly?

Also the description of the function is misleading I would say, because it is not all None value that this checks. It checks where the values are "falsy", thus [], {}, "", 0 would also return false.

```
>>> d = {"a": "a"}
>>> any(d.values())
True
>>> d = {"a": None}
>>> any(d.values())
False
>>> d = {"a": {}}
>>> any(d.values())
False
>>> d = {"a": ""}
>>> any(d.values())
False
>>> d = {"a": 0}
>>> any(d.values())
False